packages/block-library/src/image/edit.js

```diff
@@ -329,6 +331,11 @@ class ImageEdit extends Component {
 		this.setState( {
 			isEditing: ! this.state.isEditing,
 		} );
+		if ( this.state.isEditing ) {
```

Comment:
On testing the logic here is the wrong way around. When editing the 'viewing' message was read and vice versa. 

The preceding call to setState doesn't take place immediately, instead the change is queued up, so this.state.isEditing still has its previous value (https://reactjs.org/docs/react-component.html#setstate).

```suggestion
		if ( ! this.state.isEditing ) {
```